src/mame/skeleton/pullttrigger.cpp

```diff
@@ -0,0 +1,94 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/***********************************************************************************************************
+
+"Pull The Trigger", developed by the Korean company "Digital Silkroad Co. Ltd." and distributed by Covielsa.
+
+Standard PC with:
+-Elitegroup L4S5MG/651+ V5.0 motherboard.
+-512MB RAM, DDR PC2100 (one SIMM).
+-Wibu Systems WibuKey USB security dongle.
+-GeForce 4 MX440 8X 64MB DDR TV-OUT video card.
+-Exact CPU model and speed unknown.
+-Samsung SP0411N/OMD 40GB HDD.
+
+And an external PCB for inputs (guns, etc.) with:
+-ATF1502AS
+-AT89C51
+-AT89C2051
+-11.0592 MHz xtal near the AT89C2051
+-11.0592 MHz xtal near the AT89C51
+-4.9152 MHz xtal near the ATF1502AS.
+
+Also, the machine allows players to keep their scores on military dogtag like cards with a SEEPROM, named
+"DSR-PT1 MEMORY" (they were probably for sale).
+
+***********************************************************************************************************/
+
+#include "emu.h"
+#include "cpu/i386/i386.h"
+#include "machine/pci.h"
+
+
+namespace {
+
+class pullttrig_state : public driver_device
+{
+public:
+	pullttrig_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+	{ }
+
+
+	void pullttrig(machine_config &config);
+
+private:
+	required_device<cpu_device> m_maincpu;
+	void pullttrig_io(address_map &map);
+	void pullttrig_map(address_map &map);
+};
+
+
+void pullttrig_state::pullttrig_map(address_map &map)
+{
+	map(0x00000000, 0x0009ffff).ram();
+	map(0x000e0000, 0x000fffff).rom().region("bios", 0);
+	map(0xfffe0000, 0xffffffff).rom().region("bios", 0);
```

Comment:
BIOS is 0x40000 in size, should work with this arrangement:
```
	map(0x000e0000, 0x000fffff).rom().region("bios", 0x20000);
	map(0xfffc0000, 0xffffffff).rom().region("bios", 0);
```

Reply:
Done, thanks!!!